cpp/src/reshape/explode.cu

```diff
@@ -55,58 +59,72 @@ struct explode_functor {
 template <>
 std::unique_ptr<table> explode_functor::operator()<list_view>(
   table_view const& input_table,
-  size_type explode_column_idx,
+  int const explode_column_idx,
```

Comment:
We always use fixed-width types like `int32_t` in libcudf, but in this case it seems like `size_type` is more appropriate anyways. 
```suggestion
  size_type const explode_column_idx,
```